server/constants/activities.ts

```diff
@@ -90,7 +90,7 @@ enum ActivityTypes {
   ORDER_PENDING_CONTRIBUTION_REMINDER = 'order.reminder.pendingFinancialContribution',
   ORDER_PROCESSING = 'order.processing',
   ORDER_PAYMENT_FAILED = 'order.payment.failed',
-  ORDER_THANKYOU = 'order.thankyou',
+  ORDER_THANKYOU = 'order.thankyou', // never used?
```

Comment:
I can't find any entries of this activity in the db, deprecated?

Reply:
Hey @gustavlrsn, these activities are synthetically notified and are not persisted in the database:
https://github.com/opencollective/opencollective-api/blob/main/server/lib/payments.js#L795-L801

Reply:
Gotcha, thank you! Removed the comment